contracts/MintShop1155.sol

```diff
@@ -695,7 +702,7 @@ contract MintShop1155 is Sweepable, ReentrancyGuard {
 
     @param _ids An array of pool IDs to retrieve information about.
   */
-  function getPools(uint256[] calldata _ids) external view
+  function getPools(uint256[] calldata _ids, uint256 _itemId) external view
```

Comment:
nit: call this `_itemIndex` instead of `_itemId` because it's really just an index of a particular item in the `items` array instead of something that uniquely identifies an item.